src/components/HeaderText/Wavy.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+const Wavy = ({ textAlign, size }) => {
```

Comment:
We actually dont need this extra component anymore. You can simply do this by using negative margin as we discussed in meeting.

Remove this Wavy text.

Also, remove `headertext` folder too. Intsead keep `HeaderText` component directly in `component` folder.

Reply:
Hi, @rt4914 I removed `headertext `folder and removed extra Wavy component, instead added this wavy svg directly in `headerText `component. Thanks